README.md

```diff
@@ -1,3 +1,61 @@
-# DAT640-2022: Information Retrieval and Text Mining course at UiS
+# DAT640-2022: Information Retrieval and Text Mining course, University of Stavanger
 
-Materials for public release (i.e., links to slides and exercises) will be shared in this repository _after_ the semester.
+This repository contains course material (lectures, labs, and assignments) for the 2022 fall edition of the course.
+
+## Course format
+
+The course followed a standard format, with lectures, assignments and group projects. Physical lecturing led by the lecturer, included discussions, Kahoot quizzes, and hands-on exercises. The lectures were not be recorded/streamed, physical attendance was expected. Individual assignments  led by the TAs were dedicated time to work on graded assignments. Student that needed help with assignments were expected to attend in person. Group projects meetings were 15 minutes time slots dedicated for getting help/feedback on progress and ideas.
```

Comment:
- "were not be recorded/..." => "were not recorded/..."
- replace "Individual assignments ... " sentence => "Labs were led by TAs and dedicated to working on individual (graded) assignments."
- replace "Group projects meetings ..." sentence => "For the group projects, groups could book 15min weekly slots to get help/feedback on progress and ideas from the lecturer/TAs."

Reply:
Done